ui/App/ProfileScreen.svelte

```diff
@@ -70,15 +185,84 @@
     </Button>
   </Header>
 
-  <ActionBar>
-    <TabBar slot="left" tabs={tabs(activeTab)} />
-  </ActionBar>
-
-  {#if activeTab === "projects"}
-    <ProjectsTab />
-  {:else if activeTab === "following"}
-    <FollowingTab />
-  {:else}
-    {unreachable(activeTab)}
+  {#if $profileProjectsStore.status === remote.Status.Success}
+    {#if $profileProjectsStore.data.cloned.length === 0 && $profileProjectsStore.data.requests.length === 0}
+      <EmptyState
+        text="You haven't created or followed any projects yet"
+        primaryActionText="Start your first project"
+        on:primaryAction={() => modal.toggle(CreateProjectModal)}
+        secondaryActionText="Or look for a project"
+        on:secondaryAction={() => modal.toggle(SearchModal)} />
+    {:else}
+      <ul class="grid" data-cy="project-list">
+        {#each $profileProjectsStore.data.cloned as project}
+          <li>
+            <ProjectCardSquare
+              {project}
+              isMaintainer={isMaintainer(session.identity.urn, project)}
+              on:click={() => openProject({ detail: project })} />
+          </li>
+        {/each}
+        {#each $profileProjectsStore.data.follows as project}
+          <li>
+            <ProjectCardSquare
+              isMaintainer={isMaintainer(session.identity.urn, project)}
+              {project}
+              on:click={() => openProject({ detail: project })} />
+          </li>
+        {/each}
+        {#if $profileProjectsStore.data.requests.length > 0}
+          <li class="box requests">
+            <div>
+              <h2>Still looking...</h2>
+              <p style="margin-top: 1rem;">
+                {projectCountText($profileProjectsStore.data.requests.length)} you’re
+                following haven’t been found yet.
+              </p>
+            </div>
+            <Button
+              variant="outline"
+              dataCy="show-requests"
+              on:click={() => {
+                showRequests = !showRequests;
+              }}
+              style="align-self: flex-start;">
+              {!showRequests ? "Show" : "Hide"}
+              {projectCountText($profileProjectsStore.data.requests.length)}
+            </Button>
+          </li>
+          {#if showRequests}
+            {#each $profileProjectsStore.data.requests as project}
+              <li
+                class="request-card box"
+                data-cy="undiscovered-project"
+                out:fade|local={{ duration: 200 }}>
+                <CopyableIdentifier kind="radicleId" value="{project.urn}}" />
```

Comment:
```suggestion
                <CopyableIdentifier kind="radicleId" value={project.urn} />
```
<img width="449" alt="Screenshot 2021-09-28 at 08 31 25" src="https://user-images.githubusercontent.com/158411/135035047-9456f6cc-55b8-4512-a9d1-78ab71e5a75b.png">